code-challenges.js

```diff
@@ -7,62 +7,120 @@
 // --------------------INSTRUCTOR EXAMPLE: Create the code that determines which of the two strings has more characters. Use the two sets of test variables provided.
 
 // Describe your process:
+    //First create an if else statement to check both fruit lengths to determine if the     condition of having a greater length than the other is met.
+
+    //If true, expected outcome is 'banana' for ex:1, 'cherry' for ex:2.
 
 // Set one:
-const fruit1 = "apple"
-const fruit2 = "banana"
-// Expected outcome: "banana"
+// const fruit1 = "apple"
+// const fruit2 = "banana"
+// // Expected outcome: "banana"
+
+// if (fruit1.length < fruit2.length) {
+//     console.log(fruit2)
+// }
+// else if (fruit1.length > fruit2.length) {
+//     console.log(fruit1)
+// }
 
 // Set two:
 // const fruit1 = "cherry"
 // const fruit2 = "kiwi"
 // Expected outcome: "cherry"
 
+// if (fruit1.length > fruit2.length) {
+//     console.log(fruit1)
+// }
+// else if (fruit1.length < fruit2.length) {
+//     console.log(fruit2)
+// }
+
 // --------------------1) Create the code that determines if a given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
 // Describe your process:
+    //First create an if else statement to check if const temp is below 212, above 212, or = to 212.
 
-const temp = 42
+    //If below 212, console log `${temp} is below boiling point`
+    //else if above 212, console log `${temp} is above boiling point`
```

Comment:
I see in your pseudocode to use an else if statement for the other statements, but I only see if statements below.  I think you were heading in the right direction with your pseudocode as else if will allow you to run any of these conditions at a given time.